core/chains/evm/txmgr/orm.go

```diff
@@ -42,6 +42,78 @@ type orm struct {
 
 var _ ORM = (*orm)(nil)
 
+// Structs saved in database should be internal to
+// the ORM and distinct from generic params
```

Comment:
This comment is relevant as a PR comment, about why we are doing this.

But not for code comment.

In code, just mention that this struct mirrors the DB schema for table eth_receipt. DONT touch this struct unless also modifying the DB schema.